packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -49,8 +57,17 @@ export const createDefaultChatHandlers = memoizeOne(
         };
         await chatThreadClient.sendMessage(sendMessageRequest, options);
       },
-      onUpdateMessage: async (messageId: string, content: string, metadata?: Record<string, string>) => {
-        await chatThreadClient.updateMessage(messageId, { content, metadata });
+      onUpdateMessage: async (
```

Comment:
Ouch!

We diverged from the underlying headless chat SDK for this method. `content` and `metadata` should be part of the `options` bag :(